src/Jackett.Common/Definitions/divteam.yml

```diff
@@ -120,15 +120,15 @@ search:
       optional: true
       filters:
         - name: append
-          args: " English"
+          args: " English-DivTeam"
```

Comment:
Like with https://github.com/Jackett/Jackett/pull/10752 the tracker name does not need to be added